rotkehlchen/accounting/transactions.py

```diff
@@ -60,11 +60,18 @@ def process(
         type_identifier = event.get_type_identifier()
         event_settings = self.tx_event_settings.get(type_identifier, None)
         if event_settings is None:
-            log.debug(
-                f'During transaction accounting found transaction event {event} '
-                f'with no mapped event settings. Skipping...',
+            event_settings = self.tx_event_settings.get(
+                event.get_type_identifier(include_counterparty=False),
             )
-            return 1
+            if (
+                event_settings is None or
+                event_settings.accounting_treatment != TxAccountingTreatment.SWAP  # for swaps we fallback to default treatment  # noqa: E501
+            ):
```

Comment:
1. Looks better this way
2. What do you mean exactly?

Don't you want to say something more like: "For swaps we we have a default treatment but for the rest we bail"

```suggestion
            if (    # for swaps we fallback to default treatment
                event_settings is None or
                event_settings.accounting_treatment != TxAccountingTreatment.SWAP
            ):
```

Reply:
Yes